features/logging/logging_acceptance.feature

```diff
@@ -0,0 +1,163 @@
+@clusterlogging
+@commonlogging
+Feature: Logging smoke test case
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-37508
+  @admin
+  @destructive
+  Scenario: One logging acceptance case for all cluster
+# Deploy cluster-logging operator via web console
+    Given logging service is removed successfully	
+    Given the logging operators are redeployed after scenario	
+    Given logging channel name is stored in the :logging_channel clipboard	
+    Given I register clean-up steps:	
+    """	
+    Given logging service is removed successfully	
+    Then the step should succeed	
+    """	
+    Given "cluster-logging" packagemanifest's catalog source name is stored in the :clo_opsrc clipboard	
+    Given "elasticsearch-operator" packagemanifest's catalog source name is stored in the :eo_opsrc clipboard	
+
+    Given I switch to the first user	
+    Given the first user is cluster-admin	
+    Given I open admin console in a browser	
+    # subscribe cluster-logging-operator	
+    When I perform the :goto_operator_subscription_page web action with:	
+      | package_name     | cluster-logging     |	
+      | catalog_name     | <%= cb.clo_opsrc %> |	
+      | target_namespace | openshift-logging   |	
+    Then the step should succeed	
+    And I perform the :set_custom_channel_and_subscribe web action with:	
+      | update_channel    | <%= cb.logging_channel %> |	
+      | install_mode      | OwnNamespace              |	
+      | approval_strategy | Automatic                 |	
+    Given cluster logging operator is ready	
+    # subscribe elasticsearch-operator
+    When I perform the :goto_operator_subscription_page web action with:	
+      | package_name     | elasticsearch-operator        |	
+      | catalog_name     | <%= cb.eo_opsrc %>            |	
+      | target_namespace | openshift-operators-redhat    |	
+    Then the step should succeed	
+    When I perform the :set_custom_channel_and_subscribe web action with:	
+      | update_channel    | <%= cb.logging_channel %> |	
+      | install_mode      | AllNamespace              |	
+      | approval_strategy | Automatic                 |	
+    Then the step should succeed
+    And I close the current browser
+# ES Metrics
+    Given I wait for the "monitor-elasticsearch-cluster" service_monitor to appear
```

Comment:
Before checking the servicemonitor, do you create the clusterlogging/instance? Seems no. You call `logging service is removed successfully	` in line 11, this step will delete `openshift-logging` and `openshift-operators-redhat` project, everything is deleted before you subscribe the operators,  so here we should create the clusterlogging/instance.